commonui/src/main/java/com/hyperwallet/android/ui/common/util/CurrencyParser.java

```diff
@@ -127,4 +127,22 @@ public CurrencyDetails getCurrency(String currencyCode) {
         }
         return null;
     }
+
+    public static String getRateWithFourDecimal(String rate) {
```

Comment:
- number of decimal places should be a parameter, why hard code?
- Method name should be generic enough, we could use this method to format any amount for any number of decimal places then instead of just Rate

Reply:
updated to generic method